driver-kotlin-sync/src/main/kotlin/com/mongodb/kotlin/client/FindIterable.kt

```diff
@@ -53,6 +55,7 @@ public class FindIterable<T : Any>(private val wrapped: JFindIterable<T>) : Mong
      * @return this
      * @since CSOT
      */
+    @Alpha(Reason.CLIENT)
```

Comment:
Shuldn't `com.mongodb.client.FindIterable.timeoutMode` also be marked as `Alpha`? If yes, could you please check again the rest of the codebase, as maybe there are other misses.